README.md

```diff
@@ -1,2 +1,65 @@
-# ufs-coastal-app
-UFS Coastal Application
+# Introduction 
+The ufs-coastal-app repository is the umbrella for the UFS Coastal project currently under development by NOAA and NCAR, which supports coastal forecast requirements. The UFS Coastal Application is meant to constitute a workflow for coastal forecasting, wherein the coupling infrastructure is provided by [this fork of  ufs-weather-model](https://github.com/oceanmodeling/ufs-weather-model), which is currently in active development.
+
+The [model-level repo](https://github.com/oceanmodeling/ufs-weather-model) contains the model code and external links needed to build the Unified Forecast System (UFS) coastal model executable and associated model components, including the ROMS, FVCOM, ADCIRC and SCHISM plus WaveWatch III model components. 
+
+Here, model component means a forecast component (e.g. coastal ocean model) that represents a physical domain (e.g. Atlantic Ocean) or process (e.g. biogeochemistry). An application is a workflow that is designed for a particular forecasting purpose. A workflow consists of a set of model components and coupling infrastructure, provided by UFS Coastal Model, pre- and post-processing scripts, and is associated with a range of valid coupling configurations. 
+
+# Where to find more information 
+[Documentation](https://oceanmodeling.github.io/ufs-coastal-app/versions/main/html/index.html)  (in development)
+[UFS Coastal Wiki](https://github.com/oceanmodeling/ufs-coastal-model/wiki/) 
+[UFS Coastal Discussions Board](https://github.com/oceanmodeling/ufs-coastal/discussions)
+[UFS Community Discussions](https://github.com/orgs/ufs-community/discussions)
+[UFS Community Wiki](https://github.com/ufs-community/ufs/wiki)
+[Contributing code to the UFS (EPIC)](https://github.com/ufs-community/ufs/wiki)
+
+# Points of contact
+Points of contact & repos for specific questions regarding each model component, as well as NOAA and NCAR project teams, are listed below. 
+
+[ADCIRC](https://github.com/adcirc/adcirc)
```

Comment:
In here, do we need to point our forks. Otherwise, users will open issue related with coastal application in the authoritative repositories.

Reply:
@saeed-moghimi-noaa reviewed this at one point and suggested to pointing to original model repos instead. But you make a good point

Reply:
@janahaddad maybe we could also explicitly add forks and direct users to open an issue in the forks rather then authoritative repository if they are thinking that the issue is related with the coastal app. I think model level issues need to go authoritative repository but I am not sure model developers are able to reproduce the issue in their side under ufs-coastal (now ufs-weather-model). This was an issue with ROMS. they are not using ufs-coastal to compile their configuration and using custom script.

Reply:
That's basically what I did. These are my first ever pull requests so hopefully I did this correctly!